model/Licensing/Licensing.md

```diff
@@ -4,7 +4,7 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-The Licensing Profile defines a minimum set of license information required to comply with typical license compliance use cases.
+The Licensing Profile defines a minimum set of license information to facilitate compliance with typical license  use cases.
```

Comment:
With "compliance" removed, I think there's now two spaces between "license" and "use cases" -- probably remove the stray space too?